models/eprop_iaf.h

```diff
@@ -62,20 +62,25 @@ E-prop plasticity was originally introduced and implemented in TensorFlow in [1]
 The membrane voltage time course :math:`v_j^t` of the neuron :math:`j` is given by:
 
 .. math::
-    v_j^t &= \alpha v_j^{t-1}+\sum_{i \neq j}W_{ji}^\mathrm{rec}z_i^{t-1}
-             + \sum_i W_{ji}^\mathrm{in}x_i^t-z_j^{t-1}v_\mathrm{th} \,, \\
-    \alpha &= e^{-\frac{\Delta t}{\tau_\mathrm{m}}} \,,
-
-whereby :math:`W_{ji}^\mathrm{rec}` and :math:`W_{ji}^\mathrm{in}` are the recurrent and
-input synaptic weights, and :math:`z_i^{t-1}` and :math:`x_i^t` are the
+  v_j^t &= \alpha v_j^{t-1}+ \zeta \left( \sum_{i \neq j} W_{ji}^\text{rec} z_i^{t-1}
+    + \sum_i W_{ji}^\text{in} x_i^t \right) -z_j^{t-1} v_\text{th} \,, \\
+  \alpha &= e^{ -\frac{ \Delta t }{ \tau_\text{m} } } \,, \\
+  \zeta &=
+    \begin{cases}
+      1 \\
+      1 - \alpha
+    \end{cases} \,, \\
+
+whereby :math:`W_{ji}^\text{rec}` and :math:`W_{ji}^\text{in}` are the recurrent and
+input synaptic weight matrices, and :math:`z_i^{t-1}` and :math:`x_i^t` are the
 recurrent and input presynaptic spike state variables, respectively.
 
```

Comment:
```suggestion
input synaptic weight matrices, respectively, and :math:`z_i^{t-1}` is the
recurrent presynaptic spike state variable,  while :math:`x_i^t` represents the 
input presynaptic spike at time t.
```
$x_i^t$ is not a state variable

Reply:
I agree that $x_i^t$ is not a state variable. I feel, the formulation "represents the input presynaptic spike at time t" might be misunderstood that $x_i^t$ only contains the spike times. To express that it is 0 if there is no spike and 1 if there is a spike, maybe we can say 

"(...) while :math:`x_i^t` represents the input at time $t$." 

*presynaptic* is not necessary here, since *input*  contains that piece of information already.

Reply:
@akorgor That's true and I agree. Plus is less text.

Reply:
Fixed in https://github.com/JesusEV/nest-simulator/pull/16/commits/63f548e6d8457953768bbec9f903ce46fb2ecc46